js/utils/musicutils.js

```diff
@@ -3824,7 +3824,7 @@ function getNote(
     let note;
     let articulation;
 
-    if (temperament === "equal") {
+    if (temperament = "equal") {
```

Comment:
This cannot be correct. You are making an assignment to temperament. Maybe you want .startsWith?

Reply:
But I thought it's not returning anything for this case as it's just checking and assigning the value for temperament to be equal again we had used this pattern many times in the files above so I don't think so it might be needed change there but I ll try and get back to you @walterbender